sql/core/src/test/resources/sql-tests/results/string-functions.sql.out

```diff
@@ -1,5 +1,5 @@
 -- Automatically generated by SQLQueryTestSuite
--- Number of queries: 142
+-- Number of queries: 143
```

Comment:
hmm, only increased the number?

Reply:
This is the generated results, and did a check that 143 is the right number